docs/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Quick description
+
+Please include a summary of the changes and the related issue or the improvement. 
+Please also include relevant motivation and context. List any dependencies that are required for this change if needed.
+
+...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## Provide data and command line to test
+
+...
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project (mostly pep8 compliant)
```

Comment:
```suggestion
- [ ] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
```

Reply:
We could replace the link with scil documentation (linting with autopep8 should be described there if it is not already)

Reply:
Let's use this link for now and will improve later.

Reply:
I can put this on my to-do list.